data_architect_misc/data_transformer/transform_functions/aed_russia_transform_functions.py

```diff
@@ -28,4 +29,17 @@ def implement_country_specific_transform_functions_in_this_file_like_this(
         """Insert description here."""
         return df
 
+    def rename_HARMONIZED_CATEGORY_data_using_dictionary_values(
+            self,
+            df
+    ):
+        # As of July 8, 2020, Phyo decided that we will map multi-category advertisements for Russia into OC for now
+        NON_STANDARD_HARMONIZED_CATEGORY = {
```

Comment:
Don Maicol,

We don't need to create a constant (the variable with CAPITAL_LETTERS) unless we plan to reuse it in other methods. So it's better to name the variable like this: 'non_standard_harmonized_category'. Even better, we can name it like this: 'temporary_harmonized_category_mappings_for_russia'. That will explain code reader why they are used. Hope that makes sense. :)